src/java/org/apache/cassandra/locator/ReplicaPlans.java

```diff
@@ -41,6 +43,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import org.apache.cassandra.config.CassandraRelevantProperties;
```

Comment:
oh, i see.   these imports are a real mess already… 😢 
the correct thing to do is to put this line after line 49 (with the other cassandra import lines)

@djatnieks what's happened with these imports ? 
trunk and branches going back looked [fine](https://github.com/apache/cassandra/blob/trunk/src/java/org/apache/cassandra/locator/ReplicaPlans.java),  is this something in the CC rebasing …?

Reply:
The current `vsearch` branch is an extension of CC4. I know that the CC4 imports are not the same as Apache C*, but don't know exactly why. And the answer can vary by file. However, at this point, it's not related to the rebasing on Apache C* 5.0 - that will be on a different branch.

At least in 5.0 the normal ant target flags unused and duplicate imports and I have been cleaning that up, but not necessarily the ordering.